portfolio/pom.xml

```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```

Comment:
nit: Remove TODO now that it's set. I probably forgot to do this on my own repo.